src/mo_types/type.ml

```diff
@@ -16,10 +16,11 @@ type obj_sort =
  | Module
```

Comment:
I think you also need to adjust the lub and glb computation on types to account for the new subtyping between stable and local functions (to match your changes to subtyping)

for example
```
fun l() {};
persistent func p() {};
let g = if b l else p;
```
Typing g requires you to take the lub of the local and persistent function types of  `l` and `p`. You code will probably produce Any for g rather that (*local*) () -> ()